department-of-veterans-affairs/caseflow
# Clear Document Content Search Bar Results

Resolves [Clear Document Content Search Bar Results](https://jira.devops.va.gov/browse/APPEALS-31912)

# Description
These changes add functionality to the document content searchbar, that adds a X button within the search bar. When clicked, this X button will reset the search term inside of the searchbar and undo any filtering caused by the document content search. It will not affect other filter types.

## Acceptance Criteria

## Testing Plan

1. Navigate to caseflow and sign in as a user who can access reader (BVADWISE works)
2. Navigate to the reader section of caseflow
3. Enter a search term into the document contents area and hit search. Notice that the list of documents has now changed.
4. Click the X button within the searchbar, and notice that the table has reset all filtering related to document content search, and also cleared out the searchterm.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="1004" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/2f2b4a64-189e-4942-a707-0edfe8a24666">
|<img width="994" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/69e42c31-5aba-4c7b-9508-ac5d357c1bdb">
